DAEValidator/CMakeLists.txt

```diff
@@ -85,9 +85,9 @@ if (NOT LIBXML2_FOUND)
 endif ()
 
 if (WIN32)
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP /Wall /WX /wd4505 /wd4514 /wd4668 /wd4710 /wd4711 /wd4820")
 else ()
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Werror")
```

Comment:
Always a good thing even if it requires more work on our part, see previous comment